src/generator.rs

```diff
@@ -56,31 +45,23 @@ impl Generator {
     pub const TONIC_INCLUDE_FILE: &str = "mod.rs";
 
     /// Run the generator for a dependency and output files into `out`
-    pub async fn run(&self, config: GenerationConfig<'_, '_>) -> eyre::Result<()> {
+    pub async fn run(&self, output: &Path) -> eyre::Result<()> {
         let protoc = protoc_bin_path().wrap_err("Unable to locate vendored protoc")?;
 
         std::env::set_var("PROTOC", protoc.clone());
 
-        let out = config.out.join(config.package.as_str());
-
-        fs::remove_dir_all(&out).await.ok();
-
-        fs::create_dir_all(&out)
-            .await
-            .wrap_err("Failed to recreate dependency output directory")?;
+        let store = Path::new(PackageStore::PROTO_PATH);
+        let protos = PackageStore::collect(store).await;
+        let includes = &[store];
 
         match self {
             Generator::Tonic => {
-                let protos = PackageStore::collect(config.location).await;
-
-                let includes = &[&config.location];
-
                 tonic_build::configure()
                     .build_client(true)
                     .build_server(true)
                     .build_transport(true)
                     .compile_well_known_types(true)
-                    .out_dir(&out)
+                    .out_dir(output)
```

Comment:
if `output` is the output directory, and it is set via `.out_dir(...)`, i would probably simply name it `out_dir`